assets/components/form/SymfonyForm.tsx

```
@@ -0,0 +1,147 @@
+import {Label} from '@/components/ui/label';
+
+type SymfonyFormProps = {
+    formName: any;
+    // submitUrl: string;
+    // initialData: Record<string, any>;
+    // onSuccess?: (response: any) => void;
+    // onError?: (error: any) => void;
+    // elements?: React.ReactNode[];
+}
+
+export function SymfonyFormWrapper({}: SymfonyFormProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)